app/workers/emails_projects_export_worker.rb

```diff
@@ -0,0 +1,27 @@
+require 'csv'
+
+class EmailsProjectsExportWorker
+  include Sidekiq::Worker
+
+  sidekiq_options queue: :data_low
+
+  def perform(project_id)
+    @project = Project.find(project_id)
+
+    # TODO: use zoo wide email export media resources once bucket paths
```

Comment:
Can you move this comment to the `DirectToS3` class as it's only there since we can't use the medium resource to put these files to a different bucket path.